app/views/profiles/_generate_key.html.erb

```diff
@@ -0,0 +1,3 @@
+<div class="medium-4-columns">
+  <button type="sumbit" class="button">Key Button</button>
```

Comment:
This should probably read something more like "Get a New Key" to match what's in account management now.